data/upgrades.js

```diff
@@ -1800,6 +1810,18 @@
     "xws": "heavyscykinterceptor",
     "grants": [
       {
+        "type": "slot",
+        "name": "Cannon"
+      },
+      {
+        "type": "slot",
+        "name": "Torpedo"
+      },
+      {
+        "type": "slot",
+        "name": "Missile"
+      },
+      {
```

Comment:
This is technically not correct. You will get _one_ of the slots, not all of them. I think we should have a way of defining a choice of slots. But until then, could you remove the `grants` field for this card? I'm happy to merge the rest of the changes :)